src/cantools/database/can/formats/sym.py

```diff
@@ -775,14 +775,19 @@ def _get_enum_name(signal: Signal) -> str:
 def _dump_choice(signal: Signal) -> str:
     # Example:
     # Enum=DPF_Actv_Options(0="notActive", 1="active", 2="rgnrtnNddAtmtcllyInttdActvRgnrt", 3="notAvailable")
+    nl = '\n'
+
     if not signal.choices:
         return ''
 
     enum_str = f'Enum={_get_enum_name(signal)}('
-    is_first_choice = True
-    for choice_number, choice_value in signal.choices.items():
-        enum_str += f'{", " if not is_first_choice else ""}{choice_number}="{choice_value}"'
-        is_first_choice = False
+    for choice_count, (choice_number, choice_value) in enumerate(signal.choices.items()):
+        if choice_count % 10 == 0 and not choice_count == 0:
```

Comment:
better use the unequals operator:
```suggestion
        if choice_count % 10 == 0 and choice_count != 0:
```